topiary-cli/src/io.rs

```diff
@@ -150,6 +150,13 @@ impl InputFile<'_> {
     }
 }
 
+/// Simple helper function to read the full content of an io Read stream
+pub(crate) fn read_input(input: &mut dyn io::Read) -> Result<String> {
+    let mut content = String::new();
+    input.read_to_string(&mut content)?;
+    Ok(content)
+}
```

Comment:
Similar to the `read_input` below but with cli errors:
https://github.com/mkatychev/topiary/blob/3ea93395bd6f3a06cd308295bc02f64e40e80983/topiary-core/src/lib.rs#L293-L297